exotica/src/Tools.cpp

```diff
@@ -37,6 +37,8 @@
 #include <iostream>
 #include <typeinfo> //!< The RTTI Functionality of C++
 #include <cxxabi.h> //!< The demangler for gcc... this makes this system dependent!
+#include <regex>
```

Comment:
Good observation.
Let's keep moving towards 16.04 but make a list of dependencies for now.